.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: AOCkWPrF6DMcsxEKZ6iFRqrfgOBJ1WAmv
```

Comment:
Set this as an environment variable `COVERALLS_REPO_TOKEN`
See [docs](https://github.com/nickmerwin/node-coveralls#running-locally).